src/cloud/social/provider.ts

```diff
@@ -58,6 +58,14 @@ interface SavedContact {
   description?: string;
 }
 
+var UserStatus = {
```

Comment:
Great, thanks for changing this.

Regarding moving this code to the uProxy repo, it seems like most of the changes we're making  here are in response to new UI and it's generally a pain to test changes cross-repo and release uproxy-lib just for cloud social provider changes (my dream: one giant repo). Anyway, it's not pressing so just food for thought for now.